src/RemoteAPI/content-suggestions/class-suggest-linked-reference-api.php

```diff
@@ -73,6 +73,7 @@ public function get_links(
 		// Convert the links to Smart_Link objects.
 		$links = array();
 		foreach ( $decoded->result as $link ) {
+			$link     = apply_filters( 'wp_parsely_suggest_linked_reference_link', $link );
```

Comment:
Do you think we should leave this in production code?

Reply:
I think so. While right now it isn't used for anything other than our testing, I think it might be useful down the line if we need to filter something on each suggested link.